x/posts/internal/types/querier.go

```diff
@@ -14,7 +14,7 @@ type QueryPostsParams struct {
 	SortBy    string // Field that should determine the sorting
 	SortOrder string // Either ascending or descending
 
-	ParentID       *PostID
+	ParentID       PostID
```

Comment:
Why has this been changed to `PostID`? Couldn't we leave it `*PostID` for better understanding?